src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -108,7 +122,10 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
   }
 
   ngOnInit() {
-    this.loadUserHomeDirectory();
+    // this.loadUserHomeDirectory();
```

Comment:
This should be an input binding in the File Explorer instead of commented out
for ex:
```
<zlux-file-explorer
     [loadHomeDirOnInit]=false >
</zlux-file-explorer>
```